package.json

```diff
@@ -3,17 +3,26 @@
   "version": "0.0.1",
   "description": "CMC CCD Domain and Definitions",
   "scripts": {
+    "start": "node index.js",
+    "generate-excel": "./bin/pull-definition-from-docker.sh ${ENVIRONMENT:-local}",
```

Comment:
just moved it out and changed it hence it probably is seeing it as deleted instead of moved.
https://github.com/hmcts/cmc-ccd-domain/pull/634/files#diff-cff92b241580a73aff4c61e1ad5d2302da5d8ea0c5ae8dac98561386ac0067e6R1